tests/test_config_generation.py

```diff
@@ -34,6 +34,8 @@ def _test_configs(folder_name, **kwargs):
         _assert_yaml_objects(deployments, path.join(app_dir, 'k8s_deployments.yml'))
     if jobs:
         _assert_yaml_objects(jobs, path.join(app_dir, 'k8s_jobs.yml'))
+    if cronjobs:
+        _assert_yaml_objects(cronjobs, path.join(app_dir, 'k8s_cronjobs.yml'))
```

Comment:
I added a cronjob inside the following file `tests/configs/basic_app/kubetools.yml`. I also corrected the file `tests/configs/basic_app/k8s_cronjobs.yml`.

Here is the comment link: https://github.com/EDITD/kubetools/pull/118/commits/df9cad1fd598ed3298f0851350bb9b08507f5570

Reply:
* df9cad1 covers the first point. Adding a comment on the test file
* the second point is not covered yet

Reply:
For the second point, I don't really understand because we are comparing the objects (services, deployments, jobs, cronjobs) with their respective config files

So if the object doesn't exist, this won't work.

We have to check with `and` and not `or`: `if cronjobs and os.path.isfile(path.join(app_dir, 'k8s_cronjobs.yml'))`

To be sure that both exist, do you agree ?

Reply:
Done, here is the commit link: https://github.com/EDITD/kubetools/pull/118/commits/035611dd5c8def476ca9f45f72dba68a9f41755e

Reply:
Awesome! Thank you!